packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -28,26 +26,120 @@ const protocolVersions = ["^0.4.0", "^0.3.0", "^0.2.0", "^0.1.0"];
 const socketReferenceBufferTime = 2000;
 
 class SocketReference {
-    public references: number = 1;
-    public delayDeleteTimeout?: NodeJS.Timeout;
-    public delayDeleteTimeoutSetTime?: number;
+    private references: number = 1;
+    private delayDeleteTimeout?: NodeJS.Timeout;
+    private delayDeleteTimeoutSetTime?: number;
+    private _socket: SocketIOClient.Socket | undefined;
+    private inUse = false;
```

Comment:
This is communicating a concept more complex concept than just being "in use" - is there maybe a more descriptive name (or a comment here as opposed to in the `disconnected()` getter)?  Something like `hasEverConnected` maybe?

Reply:
I've put a comment, but also reversed it, naming isPendingInitialConnection, as that seems to better reflect the intent